src/style.css

```diff
@@ -1,3 +1,76 @@
+:root {
+  --cyan: #00acdc
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+
+.navigation {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.navigation__logo {
+  align-self: center;
+  justify-content: center;
+  height: 40px;
+}
+
+.navigation__list {
+  display: flex;
+}
+
+ul {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+.navigation__item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.navigation__link {
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+  text-align: center;
+  display: flex;
+  align-items: center;
+  color: #000;
+  height: 60px;
```

Comment:
it's not a good practice to hardcode height for elements with content inside. use padding for your links instead

Reply:
All changes have been made except this. 
Please, tell me in more detail, how to do this. Thanks for the help!

Reply:
You can leave it like it is now.
Generally you can replace height property with 
padding: 22.5px, then you'll have same height